src/DataService/Player/Player.php

```diff
@@ -615,149 +701,173 @@ public function getAdminTags(): array
      */
-    public function setAllowFullScreen($allowFullScreen)
+    public function setAllowFullScreen(?bool $allowFullScreen)
     {
         $this->allowFullScreen = $allowFullScreen;
     }
 
     /**
-     * Returns Indicates whether the player will make a shareable link available to users.
+     * Returns indicates whether the player will make a shareable link available to users.
      *
      * @return bool
      */
-    public function getAllowGetLink(): bool
+    public function getAllowGetLink(): ?bool
     {
         return $this->allowGetLink;
     }
 
     /**
-     * Set Indicates whether the player will make a shareable link available to users.
+     * Set indicates whether the player will make a shareable link available to users.
      *
-     * @param bool
+     * @param bool $allowGetLink
      */
-    public function setAllowGetLink($allowGetLink)
+    public function setAllowGetLink(?bool $allowGetLink)
     {
         $this->allowGetLink = $allowGetLink;
     }
 
     /**
-     * Returns Indicates whether the player will make an RSS link available to users.
+     * Returns indicates whether the player will make an RSS link available to users.
      *
      * @return bool
      */
-    public function getAllowRss(): bool
+    public function getAllowRss(): ?bool
     {
         return $this->allowRss;
     }
 
     /**
-     * Set Indicates whether the player will make an RSS link available to users.
+     * Set indicates whether the player will make an RSS link available to users.
      *
-     * @param bool
+     * @param bool $allowRss
      */
-    public function setAllowRss($allowRss)
+    public function setAllowRss(?bool $allowRss)
     {
         $this->allowRss = $allowRss;
     }
 
     /**
-     * Returns Indicates whether to include the search component.
+     * Returns indicates whether to include the search component.
      *
      * @return bool
      */
-    public function getAllowSearch(): bool
+    public function getAllowSearch(): ?bool
     {
         return $this->allowSearch;
     }
 
     /**
-     * Set Indicates whether to include the search component.
+     * Set indicates whether to include the search component.
      *
-     * @param bool
+     * @param bool $allowSearch
      */
-    public function setAllowSearch($allowSearch)
+    public function setAllowSearch(?bool $allowSearch)
     {
         $this->allowSearch = $allowSearch;
     }
 
     /**
-     * Returns Indicates whether the player should always display the play overlay.
+     * Returns indicates whether the player should always display the play overlay.
      *
      * @return bool
      */
-    public function getAlwaysShowOverlay(): bool
+    public function getAlwaysShowOverlay(): ?bool
     {
         return $this->alwaysShowOverlay;
     }
 
     /**
-     * Set Indicates whether the player should always display the play overlay.
+     * Set indicates whether the player should always display the play overlay.
```

Comment:
This and the above ones sound weird (Set indicates). I guess that this was autogenerated, right? If so, I am fine with them.

Reply:
Yeah, this is straight from the mpx docs. It's poor reading there too, but I think it's OK to match so it's clear where it came from.